bundles/ch.elexis.core.jpa.entities/src/ch/elexis/core/jpa/entities/OutputLogEntity.java

```diff
@@ -0,0 +1,196 @@
+package ch.elexis.core.jpa.entities;
+
+import java.time.LocalDate;
+
+import org.eclipse.persistence.annotations.Cache;
+
+import ch.elexis.core.jpa.entities.converter.BooleanCharacterConverterSafe;
+import ch.elexis.core.jpa.entities.listener.EntityWithIdListener;
+import ch.elexis.core.model.util.ElexisIdGenerator;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+/**
+ * <!-- begin-user-doc --> JPA entity representing the OUTPUT_LOG table.
+ * 
+ * This class represents a log entry in the database and contains metadata
+ * related to various output operations. <!-- end-user-doc -->
+ */
+@Entity
+@Table(name = "OUTPUT_LOG")
+@EntityListeners(EntityWithIdListener.class)
+@Cache(expiry = 15000, alwaysRefresh = false, refreshOnlyIfNewer = true)
+public class OutputLogEntity extends AbstractEntityWithId implements EntityWithId, EntityWithDeleted {
+
+	@Id
+	@GeneratedValue(generator = "system-uuid")
+	@Column(unique = true, nullable = false, length = 25)
+	private String id = ElexisIdGenerator.generateId();
+
+	/**
+	 * Soft delete flag to mark the entity as deleted without actually removing it
+	 * from the database.
+	 */
+	@Column
+	@Convert(converter = BooleanCharacterConverterSafe.class)
+	protected boolean deleted = false;
+
+	/**
+	 * Corresponds to "ObjectType" - the type of the logged object.
+	 */
+	@Column(name = "OBJECTTYPE")
+	private String objectType;
+
+	/**
+	 * Corresponds to "ObjectID" - the unique identifier of the logged object.
+	 */
+	@Column(name = "OBJECTID")
+	private String objectId;
+
+	/**
+	 * Stores the creator ID, indicating who generated the log entry.
+	 */
+	@Column(name = "CREATORID", length = 80)
+	private String creatorId;
+
+	/**
+	 * Corresponds to "Outputter" - identifies the entity responsible for the output
+	 * process.
+	 */
+	@Column(name = "OUTPUTTER")
+	private String outputter;
+
+	/**
+	 * Corresponds to "DATE" (previously "FLD_DATE") - stores the date of the log
+	 * entry. Stored as a string, but could be changed to a Timestamp or LocalDate.
+	 */
+
+	@Column(length = 8)
+	private LocalDate datum;
+
+	/**
+	 * Corresponds to "EXTINFO" - additional metadata related to the output process.
+	 */
+	@Column(name = "EXTINFO", length = 4096)
+	private String extInfo;
```

Comment:
Bitte definition ändern wie in anderen Klassen.
<pre>
	@Lob
	protected byte[] extInfo;
</pre>